packages/plugins/AddressSearch/src/components/Results.vue

```diff
@@ -100,6 +125,18 @@ export default Vue.extend({
     },
   },
   watch: {
+    featuresAvailable(): void {
+      this.$nextTick(() => {
+        const firstElement =
+          // @ts-expect-error | Type conversion is fine here as the querySelector method is monkeyPatched in core/createMap
+          (document.querySelector('[data-app]') as ShadowRoot).getElementById(
+            'polar-plugin-address-search-results-feature-0-0'
+          )
+        if (firstElement) {
+          firstElement.focus()
+        }
+      })
+    },
```

Comment:
Due to the search running automagically on entering x characters, this may be a very annoying feature to many users with a lower typing speed or fast services. Realized this on the DISH client, where the first search is for keywords stored in the client and therefore almost instantaneous – this doesn't allow my to finish writing.

Reply:
Could this be changed to be a configurable feature?  
IIRC it was a requirement given for Meldemichel

Reply:
Can now be configured and defaults to `false` c0338281